android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/AppcoinsBillingWrapper.java

```diff
@@ -20,11 +20,14 @@
 class AppcoinsBillingWrapper implements AppcoinsBilling, Serializable {
 
   private final AppcoinsBilling appcoinsBilling;
+  private final AppCoinsPendingIntentCaller pendingIntentCaller;
   private final String walletId;
-  private long timeoutInMillis;
+  private int timeoutInMillis;
 
-  AppcoinsBillingWrapper(AppcoinsBilling appcoinsBilling, String walletId, long timeoutInMillis) {
+  AppcoinsBillingWrapper(AppcoinsBilling appcoinsBilling,
+      AppCoinsPendingIntentCaller pendingIntentCaller, String walletId, int timeoutInMillis) {
```

Comment:
We can call here AppCoinsPendingIntentCaller. instance/getInstance instead of passing by parameter